app/helpers/works_helper.rb

```diff
@@ -96,12 +96,16 @@ def marked_for_later?(work)
     reading && reading.toread?
   end
 
+  def markasread_link(work)
+    link_to ts("Mark as Read"), marktoread_work_path(work)
+  end
+
   def marktoread_link(work)
     link_to ts("Mark for Later"), marktoread_work_path(work)
   end
 
-  def markasread_link(work)
-    link_to ts("Mark as Read"), marktoread_work_path(work)
+  def removemarktoread_link(work)
```

Comment:
Why are we adding this and keeping Mark as Read rather than changing `markasread_link(work)` to use the new path?